src/main/java/mate/academy/Main.java

```diff
@@ -38,39 +40,39 @@ public static void main(String[] args) throws AuthenticationException {
         Movie movie = new Movie();
         movie.setTitle("Untouchable");
         movieService.add(movie);
-        movieService.getAll().forEach(System.out::println);
+        movieService.getAll().forEach(logger::info);
 
         CinemaHall cinemaHall = new CinemaHall();
         cinemaHall.setCapacity(50);
         cinemaHall.setDescription("For adults");
         cinemaHallService.add(cinemaHall);
-        cinemaHallService.getAll().forEach(System.out::println);
+        cinemaHallService.getAll().forEach(logger::info);
 
         MovieSession movieSession = new MovieSession();
         movieSession.setMovie(movie);
         movieSession.setCinemaHall(cinemaHall);
         movieSession.setShowTime(LocalDateTime.now());
         movieSessionService.add(movieSession);
         movieSessionService.findAvailableSessions(movie.getId(),
-                LocalDate.now()).forEach(System.out::println);
+                LocalDate.now()).forEach(logger::info);
 
         authenticationService.register("bob@gmail.com", "1111");
         authenticationService.register("alisa@gmail.com", "1111");
         User bob = authenticationService.login("bob@gmail.com", "1111");
 
         shoppingCartService.addSession(movieSession, bob);
         ShoppingCart shoppingCartBob = shoppingCartService.getByUser(bob);
-        System.out.println(shoppingCartBob);
+        logger.info(shoppingCartBob);
         shoppingCartService.clear(shoppingCartBob);
-        System.out.println(shoppingCartBob);
+        logger.info(shoppingCartBob);
 
         User alisa = authenticationService.login("alisa@gmail.com", "1111");
```

Comment:
let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block